### ISSUE TITLE
Downloading files at runtime is disabled

### ISSUE THREAD

**Issue by lavekush-t** [uber/okbuck#792]
Facing this issue while running `./buckw build <target>`

```
Downloading files at runtime is disabled, please run 'buck fetch' before your build
    When running <curl>.
```

Using buck `com.uber:okbuck:0.46.1`
Android Studio `3.2.1`
buildToolVersion = `"28.0.3"`
target = `"android-28"`

also tried with 
buildToolVersion = `"27.0.3"`
target = `"android-27"`
buck `com.uber:okbuck:0.40.1`.

and `offline` mode already disabled

**Comment by kageiit** [comment]
@raviagarwal7 maybe we need to set a okbuck.buckconfig for `downloadInBuild = true` by default and turn it off if the behavior is disabled

**Comment by kageiit** [comment]
Fix released in v0.46.2
